pyocd/target/builtin/target_MIMXRT1024xxxxx.py

```diff
@@ -0,0 +1,100 @@
+    0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000,
+    0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000,
+    0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000,
+    0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000,
+    0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000,
+    0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000,
+    0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000,
+    0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000,
+    0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000,
+    0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000,
+    0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000,
+    0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000,
+    0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000,
+    0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000,
+    0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000
+    ],
+
+    # Relative function addresses
+    'pc_init': 0x200000ab,
+    'pc_unInit': 0x20000163,
+    'pc_program_page': 0x2000019f,
+    'pc_erase_sector': 0x20000181,
+    'pc_eraseAll': 0x20000167,
+
+    'static_base' : 0x20000000 + 0x00000020 + 0x00000278,
+    'begin_stack' : 0x20000700,
+    'begin_data' : 0x20000000 + 0x1000,
+    'page_size' : 0x100,
+    'analyzer_supported' : False,
+    'analyzer_address' : 0x00000000,
+    'page_buffers' : [0x20001000, 0x20001100],   # Enable double buffering
+    'min_program_length' : 0x100,
+}
+
+class MIMXRT1024xxxxx(CoreSightTarget):
+
+    VENDOR = "NXP"
+
+    memoryMap = MemoryMap(
+        RamRegion(name="itcm",      start=0x00000000, length=0x10000), # 64 KB
+        RamRegion(name="dtcm",      start=0x20000000, length=0x10000), # 64 KB
+        RamRegion(name="ocram",     start=0x20200000, length=0x20000), # 128 KB
+        FlashRegion(name="flexspi", start=0x60000000, length=0x400000, blocksize=0x1000, is_boot_memory=True,
+            algo=FLASH_ALGO_QUADSPI, page_size=0x100),
+        RamRegion(name="semc",      start=0x80000000, length=0x1e00000, is_external=True), # external sdram
+        RamRegion(name="NCACHE_REGION",      start=0x81e00000, length=0x200000, is_external=True)
+    )
+
+    def __init__(self, link):
+        super(MIMXRT1024xxxxx, self).__init__(link, self.memoryMap)
+        # self._svd_location = SVDFile.from_builtin("MIMXRT1024.xml")
```

Comment:
Is there not an SVD file for the RT1024?

Reply:
added